data/virt_autotest/setup_dns_service.sh

```diff
@@ -0,0 +1,258 @@
+#!/bin/bash
+
+source /usr/share/qa/qa_test_virtualization/shared/standalone
+wait_script="30"
+bridgeif="br123"
+bridgeip="192.168.123.1";
+dns_domain_forward="testvirt.net";
+dns_domain_reverse="123.168.192";
+dhcpd_lease_file="/var/lib/dhcp/db/dhcpd.leases"
+setup_log_file="/var/log/virt_dns_setup.log"
+
+if [ -e ${setup_log_file} ];then
+	rm -f ${setup_log_file}
+fi
+
+zypper_cmd="zypper --non-interactive in lsb-release bridge-utils bind bind-chrootenv bind-utils"
+echo -e "${zypper_cmd} will be executed\n" | tee -a ${setup_log_file}
+${zypper_cmd}
+
+#Add repo to upgrade bash to v4.3, otherwise legacy system may not support associative array
+reposarch=`arch`
+bashrepourl="http://download.suse.de/ibs/SUSE/Updates/SLE-SERVER/12-SP4/${reposarch}/update/"
```

Comment:
I guess the reason for adding it is because that it contains the bash you needed. However adding a fixed 12sp4 update repo to any product host , like sle11, sle15 , is strongly not suggested. Especially to our official milestone test. Risk is that it may also affects rpms in its dependency chain, which makes our test target incorrect. 
Would you please try a more reasonable way, like adding the host product's relevant update repo?

Reply:
Sorry for this late comment. I did not notice it until i checked the verification run. BTW now the time for dns setup  is ~3 minutes, which is quite reasonable time now. Good job  in achieving this high efficiency.

Reply:
I can make the change if someone is not satisfied with this implementation. As far as the update repo is concerned, any product newer than 12-SP4 will not get updated anything. Only 11-SP4 or 12 with service pack lower than sp4 will get updated with the required packages. And, in theory, any updated content within this PR's scope is backward compatible. It will not affect any existing functionalities. But in light of your concern, I will make some changes with more plain bash usage.

Reply:
Done. Suggest merge if no other big concerns.

Reply:
Thanks for the timely change. Will merge now.